core/src/main/java/com/amplitude/core/Amplitude.kt

```diff
@@ -102,7 +101,7 @@ open class Amplitude internal constructor(
      * @return the Amplitude instance
      */
     @JvmOverloads
-    fun track(eventType: String, eventProperties: JSONObject? = null, options: EventOptions? = null): Amplitude {
+    fun track(eventType: String, eventProperties: MutableMap<String, Any?>? = null, options: EventOptions? = null): Amplitude {
```

Comment:
Does this need to be a `MutableMap`? Can we just use `Map`?

Reply:
Only make this MutableMap as I adjust the eventProperties in BaseEvent as that. But I guess Map makes more sense for this interface when giving another thought. I will update this to Map.

Reply:
You should be able to use the `.toMutableMap()` extension function.